### PR TITLE
Revert "Disable word wrapping for `p code` (#281)" to fix #576

### LINKED ISSUE #576: Overflow of text on mobile handsets in sass docs.
Android version - 11
Chrome version - 92.0.4515.115
Docs section - At rules > at-for

Screenshot_20210906-130240_Chrome

### PR DESCRIPTION
**Pull request by kauefr**
This reverts commit 77311c93f99c91ef8f450c269fd72a8c7456a430.

Fixes #576 by allowing word-wrap in code snippets, as commented [here](https://github.com/sass/sass-site/issues/576#issuecomment-1127088521).

### Before
before
### After
after

### CODE REVIEW

**Comment by nex3** [comment]
@jina Can we put this behind a breakpoint so that only desktop-size browsers disable word wrapping? It's legitimately valuable for code examples at large screen sizes.

File: source/assets/css/visual-design/_typography.scss [hunk 1]
```diff
@@ -179,8 +179,10 @@ code {
   font-size: ($sl-font-size--small / $sl-font-size--medium) * 1em;
   line-height: 1;
 
-  nav &,
-  p & { white-space: nowrap; }
+  nav & { white-space: nowrap; }
+  p & {
+    @include sl-breakpoint--x-large { white-space: nowrap; }
```
Comment (nex3) [inline source/assets/css/visual-design/_typography.scss:184]: I think a medium breakpoint might make more sense here. It's really mobile screens which are likely to suffer from the nowrap behavior, right?

Comment (kauefr) [reply source/assets/css/visual-design/_typography.scss:184]: The issue gets worse in lower resolutions, but in this case overflow can occur in resolutions as high as 1200px. 

For comparison, screenshots at medium (640px) and large (960px) breakpoints:

**Comment by nex3** [comment]
I'd love to find a middle ground her, possibly using a smaller font size for medium screens, but I'll leave it up to @jina to figure out the details.

**Pull request merged**